models/org_team.go

```diff
@@ -196,7 +196,11 @@ func (t *Team) RemoveRepository(repoID int64) error {
 
 // NewTeam creates a record of new team.
 // It's caller's responsibility to assign organization ID.
-func NewTeam(t *Team) error {
+func NewTeam(t *Team) (err error) {
+	if err = IsUsableUsername(t.Name); err != nil {
```

Comment:
Should probably have it's own check? Since some usernames are valid as team-names